<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgHisteditConfigDialog</class>
 <widget class="QDialog" name="HgHisteditConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histedit Configuration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Start Revision</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="defaultButton">
        <property name="toolTip">
         <string>Select to use the default revision as the base</string>
        </property>
        <property name="text">
         <string>Default Revision</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="outgoingButton">
        <property name="toolTip">
         <string>Select to edit all outgoing revisions</string>
        </property>
        <property name="text">
         <string>All Outgoing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="revisionButton">
        <property name="toolTip">
         <string>Select to use a revision as start point</string>
        </property>
        <property name="text">
         <string>Revision</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="revisionGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Revision</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="numberButton">
           <property name="toolTip">
            <string>Select to specify a revision by number</string>
           </property>
           <property name="text">
            <string>Number</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="numberSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Enter a revision number</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="idButton">
           <property name="toolTip">
            <string>Select to specify a revision by changeset id</string>
           </property>
           <property name="text">
            <string>Id:</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="idEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Enter a changeset id</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="tagButton">
           <property name="toolTip">
            <string>Select to specify a revision by a tag</string>
           </property>
           <property name="text">
            <string>Tag:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="tagCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Enter a tag name</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="branchButton">
           <property name="toolTip">
            <string>Select to specify a revision by a branch</string>
           </property>
           <property name="text">
            <string>Branch:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QComboBox" name="branchCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Enter a branch name</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="bookmarkButton">
           <property name="toolTip">
            <string>Select to specify a revision by a bookmark</string>
           </property>
           <property name="text">
            <string>Bookmark:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="bookmarkCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Enter a bookmark name</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="forceCheckBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Select to force outgoing</string>
     </property>
     <property name="text">
      <string>Force outgoing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="keepCheckBox">
     <property name="toolTip">
      <string>Select to keep old nodes</string>
     </property>
     <property name="text">
      <string>Keep old nodes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>defaultButton</tabstop>
  <tabstop>outgoingButton</tabstop>
  <tabstop>revisionButton</tabstop>
  <tabstop>numberButton</tabstop>
  <tabstop>numberSpinBox</tabstop>
  <tabstop>idButton</tabstop>
  <tabstop>idEdit</tabstop>
  <tabstop>tagButton</tabstop>
  <tabstop>tagCombo</tabstop>
  <tabstop>branchButton</tabstop>
  <tabstop>branchCombo</tabstop>
  <tabstop>bookmarkButton</tabstop>
  <tabstop>bookmarkCombo</tabstop>
  <tabstop>forceCheckBox</tabstop>
  <tabstop>keepCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgHisteditConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgHisteditConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>revisionButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>revisionGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>numberSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>idEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tagButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>tagCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branchButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>branchCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarkButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmarkCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outgoingButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>forceCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
